gabbi/tests/gabbits_intercept/utf8.txt

```diff
@@ -0,0 +1,222 @@
+<h1>Unicode Demo</h1>
+
+<p>Taken from <a
+href="http://www.cl.cam.ac.uk/~mgk25/ucs/examples/UTF-8-demo.txt">http://www.cl.cam.ac.uk/~mgk25/ucs/examples/UTF-8-demo.txt</a></p>
+
+<pre>
+
+$ENVIRON['HOME']
```

Comment:
I felt obliged to check: it is documented in [substitution](http://gabbi.readthedocs.io/en/latest/format.html#substitution) but glancingly. `data` is included in the list of fields where the variables may be used. That could probably be improved.